tests/fxl-orientation_default/META-INF/container.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container version="1.0" xmlns="urn:oasis:names:tc:opendocument:xmlns:container">
+   <rootfiles>

```

Comment:
Nit: this is an old copy of the template that has 3-space indentation in the container. Mind fixing to 2-space throughout?